instances/unitOfMeasurement/kilobyte.jsonld

```diff
@@ -10,5 +10,5 @@
   "knowledgeSpaceLink": null,
   "name": "kilobyte",
   "preferredOntologyIdentifier": null,
-  "synonym": null
+  "synonym": ["KB"]
```

Comment:
I think we should specify somewhere whether 1 kilobyte is 1000 bytes (1 kB) or 1024 bytes (1 KB), and similarly for MB, etc. I wonder if we should explicitly use kibibyte (KiB) for 1024 bytes?

Reply:
@olinux is going to like you very much for this comment. Seems like everything is pointing to an extension for the unitOfMeasurement schema with properties describing:
- how units can be converted (e.g. msec to sec to min)
- what type of unit they are (e.g. time, size, volume, etc.)
- whether it is a postfix or prefix (e.g. embryonic day as E11.5 vs. day as 11.5 d) 
- which symbol should be used (instead of the workaround to add it under synonyms & with plurals defined if needed?) 

@lzehl, any thoughts on this?

Reply:
the 1000 vs 1024 bytes question would probably be disambiguated by the `knowledgeSpaceLink` or `preferredOntologyIdentifier`, e.g.:

kilobyte (1000 bytes): http://purl.obolibrary.org/obo/UO_0000234
kibibyte (1024 bytes): http://purl.obolibrary.org/obo/UO_0000245

Reply:
As for kilobytes vs kibibytes: both should be units we should have registered (please add kibibytes if they are missing from the controlledTerms list for measurement units).  

For general conversion between units: if we stick to the official names / abbreviations there are tools out there that deal only with unit conversions (time is for me a special issue, though; @apdavison did you ever deal with time conversion? (year -> months -> days; or the other way around). I would rather support the convention used by one or a selection of tools that do correct conversions than inventing a schema that provides information on how to convert in a machine-readable way. There speaks nothing against extending the definitions with such information, though.

There speaks also nothing against extending the controlledTerm schema for UnitOfMeasurement with additional properties. For example, "preferredSymbol" (for openMINDS preference; which is different to synonyms which could list all other stuff), "placement" (with selection prefix/postfix). Plurals should be defined elsewhere (e.g. in ontologies) or we just ignore them (because for display I would anyway only use the preferred symbol which does not have a plural).